packages/components/src/primitives/block-quotation/index.native.js

```diff
@@ -2,15 +2,25 @@
  * External dependencies
  */
 import { View } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement } from '@wordpress/element';
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
 export const BlockQuotation = ( props ) => {
+	const newChildren = Children.map( props.children, ( child ) => {
+		if ( child && child.props.identifier === 'citation' ) {
```

Comment:
I'm curious about why you need to test for `child`, is there any chance of that being null/undefined?